src/Components/TodoContainer.jsx

```diff
@@ -0,0 +1,195 @@
+import React from "react";
+import TodoList from "./TodoList";
+import AddTodoForm from "./AddTodoForm";
+import styles from "./TodoContainer.module.css";
+
+function TodoContainer() {
+  const [todoList, setTodoList] = React.useState([]);
+  const [isLoading, setIsLoading] = React.useState(true);
+  const [isAscending, setIsAscending] = React.useState(false);
+  const [isNewestFirst, setIsNewestFirst] = React.useState(false);
+
+  //Fetching data
+  async function fetchData() {
+    const url = `https://api.airtable.com/v0/${
+      import.meta.env.VITE_AIRTABLE_BASE_ID
+    }/${import.meta.env.VITE_TABLE_NAME}?view=Grid%20view`;
+
+    const options = {
+      method: "GET",
+      headers: {
+        Authorization: `Bearer ${import.meta.env.VITE_AIRTABLE_API_TOKEN}`,
+      },
+    };
+
+    try {
+      const response = await fetch(url, options);
+
+      if (!response.ok) {
+        const message = `Error: ${response.status}`;
+        throw new Error(message);
+      }
+
+      const data = await response.json();
+
+      const todos = data.records.map((todo) => {
+        return {
+          id: todo.id,
+          title: todo.fields.title,
+          createdTime: todo.createdTime,
+        };
+      });
+
+      setTodoList(todos);
+      setIsLoading(false);
+    } catch (error) {
+      console.log("Fetch error:", error.message);
+      return null;
+    }
+  }
+  //sort records
+  const handleSortByTitle = () => {
+    
+    const sortedTodos = [...todoList].sort((todoA, todoB) => {
+      const titleA = (todoA.title || "").toLowerCase();
```

Comment:
This function perfectly sorts letters but has some issues with sorting numbers:
<img width="652" alt="image" src="https://github.com/user-attachments/assets/b39650fc-9e64-4682-a54e-2cedbaa34991">

This is for your information, and you can try to sort letters and numbers accordingly in your final project."

Reply:
Fixed. Could you please try again? 
![Screen Shot 2024-09-28 at 11 30 18 AM](https://github.com/user-attachments/assets/40158ad8-a67e-4969-b026-bf14aeebbf4f)